### DISCUSSION TITLE
How to remove other players from result screen?

### DISCUSSION THREAD

**Discussion by AddGaming** [ppy/osu#26121] [Q&A] [answered]
Currently, other players get presented in the result screen and I wonder if there is a setting to turn it off.

![image](https://github.com/ppy/osu/assets/135885956/958951ff-3fb8-4871-b71a-0d63a8f5ae9a)

I can switch my scores in the select to only show local, so why are the results in the final screen, other players?
![image](https://github.com/ppy/osu/assets/135885956/919519b3-63fe-4b1e-9398-15743bdc25f9)

I switched it to local since other players actually give me anxiety. 
This is the only reason why I've not switched which is sad because I really like the lazer client.
I saw that the result screen is [still subject to change](https://github.com/ppy/osu/discussions/18467), so I would appreciate it, if this was considered.

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/9201.
